algorithms/kernel/dtrees/gbt/regression/gbt_regression_train_dense_default_impl.i

```diff
@@ -164,22 +164,63 @@ services::Status RegressionTrainBatchKernel<algorithmFPType, method, cpu>::compu
         DAAL_CHECK_STATUS(s, (indexedFeatures.init<algorithmFPType, cpu>(*x, &featTypes, par.splitMethod == gbt::training::inexact ? &prm : nullptr)));
     }
 
+    algorithmFPType* ptrWeightFeatImp     = nullptr;
+    algorithmFPType* ptrCoverFeatImp      = nullptr;
+    algorithmFPType* ptrCoverTotalFeatImp = nullptr;
+    algorithmFPType* ptrGainFeatImp       = nullptr;
+    algorithmFPType* ptrGainTotalFeatImp  = nullptr;
+
+    if (par.resultsToCompute & gbt::training::computeWeight)
```

Comment:
I see this "copy-past" code in regression and classification. Do we can move them in one common place

Reply:
No, we can't because different values is used for getting result values in classification and regression algorithms. But anyway I made this piece a little bit shorter